cmd/utils/nodecmd/flags_test.go

```diff
@@ -136,7 +136,7 @@ var flagsWithValues = []struct {
 		errors:      []int{},
 	},
 	{
-		flag:        "--txpool.journal-interval",
+		flag:        "--txpool.rejournal",
```

Comment:
The test should be made for both new and old flags.

Reply:
I added old flags test.
Thanks winnie~